test/redundantaudioencoder/RedundantAudioEncoder.test.ts

```diff
@@ -574,20 +574,51 @@ describe('RedundantAudioEncoder', () => {
     });
 
     it('forwards non-RED frames', () => {
+      const header = createRedHeader(
+        true,
+        encoder['opusPayloadType'],
+        0,
+        importantPayload.byteLength
+      );
+      let i = 0;
+      for (; i < 2; ++i) {
+        frame = createRTCEncodedAudioFrame(packetizationTime * i, redPayloadType, 0);
+        frame.data = createRedPayload([header], [importantPayload]);
+        encoder['senderTransform'](frame, controller);
+      }
+
+      // The next outgoing packet should not contain the previous important payloads as redundancy.
+      encoder.setNumRedundantEncodings(1);
       encoder.setRedPayloadType(redPayloadType + 1);
+      frame = createRTCEncodedAudioFrame(packetizationTime * i, redPayloadType, 0);
+      frame.data = createRedPayload([header], [importantPayload]);
       encoder['senderTransform'](frame, controller);
-      expect(frame.data.byteLength).to.equal(0);
+      expect(frame.data.byteLength).to.equal(header.byteLength + importantPayload.byteLength);
     });
 
     it('forwards frames when the primary payload cannot be determined', () => {
+      const header = createRedHeader(
```

Comment:
I went ahead and made this test case more effective as well since it is similar in nature to the previous test case.